core/src/test/java/org/elasticsearch/plugins/PluginManagerIT.java

```diff
@@ -310,11 +308,11 @@ public void testLocalPluginInstallWithBinAndConfigInAlreadyExistingConfigDir_789
         - config/dir/testdir2.txt (version1)
         - config/dir/subdir/testsubdir.txt (version2)
          */
-        Files.write(pluginDir.resolve("config").resolve("test.txt"), "version3".getBytes(StandardCharsets.UTF_8));
-        Files.write(pluginDir.resolve("config").resolve("test2.txt"), "version1".getBytes(StandardCharsets.UTF_8));
-        Files.write(pluginDir.resolve("config").resolve("dir").resolve("testdir.txt"), "version2".getBytes(StandardCharsets.UTF_8));
-        Files.write(pluginDir.resolve("config").resolve("dir").resolve("testdir2.txt"), "version1".getBytes(StandardCharsets.UTF_8));
-        Files.write(pluginDir.resolve("config").resolve("dir").resolve("subdir").resolve("testsubdir.txt"), "version2".getBytes(StandardCharsets.UTF_8));
+        Files.write(pluginDir.resolve("config").resolve("test.txt"), "version3".getBytes(java.nio.charset.StandardCharsets.UTF_8));
+        Files.write(pluginDir.resolve("config").resolve("test2.txt"), "version1".getBytes(java.nio.charset.StandardCharsets.UTF_8));
+        Files.write(pluginDir.resolve("config").resolve("dir").resolve("testdir.txt"), "version2".getBytes(java.nio.charset.StandardCharsets.UTF_8));
+        Files.write(pluginDir.resolve("config").resolve("dir").resolve("testdir2.txt"), "version1".getBytes(java.nio.charset.StandardCharsets.UTF_8));
+        Files.write(pluginDir.resolve("config").resolve("dir").resolve("subdir").resolve("testsubdir.txt"), "version2".getBytes(java.nio.charset.StandardCharsets.UTF_8));
```

Comment:
I think the full package names aren't needed here and on lines [241](https://github.com/elastic/elasticsearch/pull/13533/files#diff-1d2b3b1a5f383a7db7006439663c33daR241), [275](https://github.com/elastic/elasticsearch/pull/13533/files#diff-1d2b3b1a5f383a7db7006439663c33daR275), [277](https://github.com/elastic/elasticsearch/pull/13533/files#diff-1d2b3b1a5f383a7db7006439663c33daR277), [278](https://github.com/elastic/elasticsearch/pull/13533/files#diff-1d2b3b1a5f383a7db7006439663c33daR278), [311](https://github.com/elastic/elasticsearch/pull/13533/files#diff-1d2b3b1a5f383a7db7006439663c33daR311), [312](https://github.com/elastic/elasticsearch/pull/13533/files#diff-1d2b3b1a5f383a7db7006439663c33daR312), [313](https://github.com/elastic/elasticsearch/pull/13533/files#diff-1d2b3b1a5f383a7db7006439663c33daR313), and [314](https://github.com/elastic/elasticsearch/pull/13533/files#diff-1d2b3b1a5f383a7db7006439663c33daR314).